test/inspecti.bats

```diff
@@ -13,6 +13,6 @@ function teardown() {
 }
 
 @test "inspect image should succed contain all necessary information" {
-	run crictl inspecti quay.io/crio/redis:alpine | jq -e .status.size
-	run crictl inspecti quay.io/crio/redis:alpine | jq -e .info.imageSpec.config.Cmd
```

Comment:
The funny thing is, this was not working at all because
1. `run` ate all the output.
2. `jq` just ignore empty input.

Now it's working, but the issue 2 still remains, so the correct way to fix this is
```sh
output=$(crictl inspecti quay.io/crio/redis:alpine)
[ "$output" != "" ]
jq -e '.status.size'<<< "$output"
jq -e '.info.imageSpec.config.Cmd' <<< "$output"
```

(can and should be addressed by a separate commit; same PR is fine)

Reply:
Just tested on my machine

```console
[kir@kir-rhat selinux]$ out=$(sudo crictl inspecti quay.io/crio/redis:alpine)
[kir@kir-rhat selinux]$ jq -e '.status.size' <<<$out
"28148166"
```